cellrank/datasets/_datasets.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Maybe just tell me once why you removed all of these?

Reply:
They were only supposed to be used for Python2.

Reply:
ok, got it.